basic-examples/sub-workflow-foreach/README.md

```diff
@@ -2,29 +2,96 @@
 
 ## Workflow Description
 
-A workflow can iterate over a sub-workflow in a for-each manner by providing a list of input objects compatible with the sub-workflow's schema and specifying `kind: foreach` in the step parameters. The Arcaflow expression language is used to pass the compatible input list object to the `items` parameter of the step.
+A workflow can iterate over a sub-workflow in a for-each manner by providing a list of
+input objects compatible with the sub-workflow's schema and specifying `kind: foreach`
+in the step parameters. The Arcaflow expression language is used to pass the compatible
+input list object to the `items` parameter of the step.
```

Comment:
I wouldn't use "pass...to...parameter".  I think "specify...value" would be better.  And, I would drop "compatible" -- we already stipulated that in the previous sentence (and, I would drop "object", as "list object" is sort of confusing while I think just "list" is clear).